usps/main.py

```diff
@@ -55,55 +60,65 @@ class ErrorResponse(BaseModel):
     error: str
 
 
-USPS_UUID = "5738f577-d283-49ec-9695-32b106c049d8"
-USPS_URL = "https://cat-aii.usps.gov/"
-
-if not settings.DEBUG:
-    credentials = service_account.IDTokenCredentials.from_service_account_info(
-        json.loads(base64.b64decode(settings.USPS_SERVICE_INFO)),
-        target_audience=settings.USPS_TARGET_AUDIENCE,
+def get_token():
+    client_id = settings.USPS_CLIENT_ID
+    client_secret = settings.USPS_CLIENT_SECRET
+    access_token_url = f"{settings.USPS_URL}/accesstoken?grant_type=client_credentials"
+    auth_response = requests.post(
+        access_token_url, auth=HTTPBasicAuth(client_id, client_secret), verify=True
     )
-    credentials.refresh(Request())
-    sslcontext = ssl.create_default_context(cafile="cat-aii-root.cer")
+    if auth_response.status_code == HTTPStatus.OK:
+        app.expires = (
+            int(auth_response.json["issued_at"]) // 1000
+            + int(auth_response.json["expires_in"])
```

Comment:
It it possible for conversions with `int()` to raise a `ValueError`, so these conversions should be wrapped in a try/except.